### ISSUE TITLE
No expected result

### ISSUE THREAD

**Issue by Librajiayou** [asm/deep_spectre#2]
Hello,  I would like to ask a question about your job. When I run your code on m computer, it could not recover the correct message and show a low accuracy when training data. Is there anything I should pay attention to or do some modify when I run your code ? Thank you ! 
The result is as follows.
`Epoch 1/10
 - 11s - loss: 4.1652 - accuracy: 0.0157 - val_loss: 4.1609 - val_accuracy: 0.0133
Epoch 2/10
 - 7s - loss: 4.1594 - accuracy: 0.0161 - val_loss: 4.1625 - val_accuracy: 0.0131
Epoch 3/10
 - 9s - loss: 4.1576 - accuracy: 0.0165 - val_loss: 4.1667 - val_accuracy: 0.0134
Epoch 4/10
 - 10s - loss: 4.1558 - accuracy: 0.0166 - val_loss: 4.1859 - val_accuracy: 0.0133
Epoch 5/10
 - 9s - loss: 4.1525 - accuracy: 0.0187 - val_loss: 4.1734 - val_accuracy: 0.0176
Epoch 6/10
 - 9s - loss: 4.1116 - accuracy: 0.0295 - val_loss: 4.1176 - val_accuracy: 0.0391
Epoch 7/10
 - 7s - loss: 4.0184 - accuracy: 0.0608 - val_loss: 4.0359 - val_accuracy: 0.0656
Epoch 8/10
 - 7s - loss: 3.9256 - accuracy: 0.0816 - val_loss: 3.9817 - val_accuracy: 0.0762
Epoch 9/10
 - 6s - loss: 3.8363 - accuracy: 0.1020 - val_loss: 3.9174 - val_accuracy: 0.0997
Epoch 10/10
 - 7s - loss: 3.7480 - accuracy: 0.1224 - val_loss: 3.8503 - val_accuracy: 0.1159
The secret message is: 8he86aaicfcorQA8a8e8aqueamiYh8YT8ifra.e.
`
